installer/inventory

```diff
@@ -2,14 +2,24 @@ localhost ansible_connection=local ansible_python_interpreter="/usr/bin/env pyth
 
 [all:vars]
 
+# Use a local distribution build container image for building the AWX package
+# This is helpful if you don't want to bother installing the build-time dependencies as
+# it is taken care of already.
+# NOTE: IMPORTANT: If you are running a mininshift install, using this container might not work
+#                  if you are using certain drivers like KVM where the source tree can't be mapped
+#                  into the build container.
+#                  Thus this setting must be set to False which will trigger a local build. To view the
+#                  typical dependencies that you might need to install see:
+#                  installer/image_build/files/Dockerfile.sdist
+use_container_for_build=false
```

Comment:
You have to uncomment some stuff in order to get OpenShift installs, so the installer technically defaults to the `local_docker` role. Should this default to `true`?

Reply:
yeah, but, I do want to call it out as usable for other than minishift (it'll work for local docker). I actually wanted this to default to true